nasa/cmr-graphql
# GQL-91: Added "standardProduct" to schema

# Overview

### What is the feature?

Allow "standardProduct" to be used in query params for collections queries.

### What is the Solution?

Added "standardProduct" to the collections schema.

### What areas of the application does this impact?

src/cmr/concepts/collection.js
src/types/collection.graphql

# Testing

### Reproduction steps

- **Environment for testing:** PROD
- **Collection to test with:** N/A

1. Pick the Standard Products portal
2. Click on the export collections button and then the CSV.
3. Ensure the collection list downloads properly.
4. Then Click on the export collections button again and then JSON.
5. Ensure the collection list downloads properly.

Graphql Testing
Ensure that querying with standardProduct as a query parameter does not return errors

QUERY: 


```
query ExportCollections($params: CollectionsInput) {
  collections(params: $params) {
    cursor
    count
    items {
      provider
      shortName
      version
      title
      standardProduct
      processingLevel
      platforms
      timeStart
      timeEnd
    }
  }
}
```


PARAMS:

```
{
  "params": {
    "includeHasGranules": true,
    "limit": 10,
    "standardProduct": true
  }
}

```


### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [N/A] I have added automated tests that prove my fix is effective or that my feature works
- [N/A] New and existing unit tests pass locally with my changes
- [n/a] I have commented my code, particularly in hard-to-understand areas
- [N/A] I have made corresponding changes to the documentation
